src/main/resources/openapi/dailyList.yaml

```diff
@@ -75,7 +79,28 @@ paths:
 
         '500':
           description: Internal Server Error
+          content:
+            application/json+problem:
+              schema:
+                $ref: './problem.yaml'
 
+  /dailylists/housekeeping:
```

Comment:
Not sure we need the housekeeping part in gateway?

Reply:
this is just the dailyList.yaml from darts-api, it's not generating the endpoints here, but is used for generating the request objects for gateway calling darts. i don't think we should be changing this yaml from what is in darts-api. its like having a xsd for the system we are trying to call.

Reply:
Cool, makes sense.